domino_generator/example/ex1/ex1_noindent.html

```diff
@@ -0,0 +1 @@
+<div title="Some help Example 1." >Some Text 1 and 1.</div><span>cond1</span><ul><li>Visible item Example 1</li><li>Visible item 2 Example 1</li></ul><div title="Some help Example 1." >Some Text 1 and 1.</div><span>cond1</span><ul><li>Visible item Example 1</li><li>Visible item 2 Example 1</li></ul><div>X</div><div>X</div>
```

Comment:
Would it be hard to remove the extra space before `>`? e.g.
`<div title="Some help Example 1." >` -> `<div title="Some help Example 1.">`